runtime.Dockerfile

```diff
@@ -1,16 +1,18 @@
 FROM ubuntu:jammy
 
+ARG PREFIX=/srv/r8/monobase
+
 ENV DEBIAN_FRONTEND=noninteractive
-ENV MONOBASE_PREFIX=/srv/r8/monobase
+ENV MONOBASE_PREFIX=$PREFIX
 ENV PATH=$MONOBASE_PREFIX/bin:$PATH
 ENV TZ=Etc/UTC
 
-ADD ./apt-dependencies.txt /tmp/apt-dependencies.txt
-RUN apt-get update \
-    && apt-get install -y $(grep -v '^#' </tmp/apt-dependencies.txt) \
-    && rm -rf /var/lib/apt/lists/* /tmp/apt-dependencies.txt
+RUN --mount=type=bind,src=.,dst=/src,ro \
+    apt-get update \
+    && apt-get install -y $(grep -v '^#' </src/apt-dependencies.txt) \
+    && rm -rf /var/lib/apt/lists/*
 
-RUN --mount=type=bind,src=../../,dst=/src,ro \
+RUN --mount=type=bind,src=.,dst=/src,ro \
```

Comment:
Not sure what `../../` means, shouldn't it be `.` i.e. PWD of project root? Why did it work before?